src/Catty/ViewController/Upload/UploadViewController.swift

```diff
@@ -406,6 +427,34 @@ class UploadViewController: UIViewController, UploadCategoryViewControllerDelega
         }
     }
 
+    func fetchTags() {
+        showLoading()
+        if let uploader = uploader, let languageCode = Locale.current.languageCode {
+            uploader.fetchTags(for: languageCode) { tags, error in
+                self.loadingView?.hide()
+                if let error = error {
+                    switch error {
+                    case .unexpectedError, .timeout:
+                        AlertControllerBuilder.alert(title: kLocalizedPocketCode, message: kLocalizedErrorInternetConnection)
+                        .addDefaultAction(title: kLocalizedOK) {
+                            DispatchQueue.main.async(execute: {
+                                self.dismissView()
+                            })
+                        }.build().showWithController(self)
+                    case .invalidLanguageTag:
```

Comment:
Please note that the tags should also be displayed in case the language code does not exist on the API-side (statusCode: 404). In that case the default labels as returned by the API should be displayed.

Reply:
Yes, it does exactly the same thing along with showing a message "Incompatible Language"

Reply:
After displaying the alert, nothing happens anymore. Please also remove this alert, since it is misleading for the user (the rest of the user interface is translated, at least partly, and the user does not know that this alert is connected to the tags). In case of an unsupported language, no alert needs to be displayed and the tags should just be displayed as they are arrived from the API.